packages/toast/src/Toast.spec.tsx

```diff
@@ -26,7 +27,8 @@ describe('packages/toast', () => {
       });
 
       const toast = queryByRole('status');
-      expect(toast).not.toBeInTheDocument();
+      expect(toast).toBeVisible();
```

Comment:
Do you think we have to register this as more than a patch, since it broke our own tests?